articles/liquity/Asset Risk Assessment - bLUSD.md

```diff
@@ -0,0 +1,203 @@
+# Asset Risk Assessment bLUSD
+### Chicken Bonds are designed to generate amplified yields for the users and protocol-owned liquidity that can be used to strengthen the subjected asset.
+
+
+# Liquity protocol overview
+
+*[Resources]*
+
+*[Official Liquity Documentation](https://docs.liquity.org/)*
+
+*[Liquity: Decentralized Borrowing Platform](https://docsend.com/view/bwiczmy) (Whitepaper)*
+
+Liquity is a decentralized borrowing protocol that allows interest-free loans against ETH as collateral. Loans are paid out in LUSD and need to maintain a minimum collateral ratio (CR) of 110%. The loans are secured by a Stability Pool containing LUSD that caters the liquidations.
```

Comment:
`Loans are paid out in LUSD and need to maintain and min CR`: please rephrase sentence. Who needs to maintain the CR: loan or the debtor?

Finally: some indication here is needed. is the CR fixed? can it be changed? Do users pay interest upon minting or is it truly a free to use protocol where the user pays absolutely nothing, mints stablecoins, pays no interest?

Reply:
The following things were made clear (lines 13-18)
- Who needs to maintain the CR: the loan or the debtor?
- Is the CR fixed?
- Borrow and redemption fee.
- Maximum expected loss on immediate liquidation of a trove.